<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Driver Loader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/myappico.ico</normaloff>:/images/myappico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>416</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="application.qrc">:/images/driver-loader.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="clopyRightlbl">
           <property name="text">
            <string>@maldevel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="addrlbl">
           <property name="text">
            <string>https://github.com/maldevel/driver-loader</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="titlelbl">
           <property name="text">
            <string>GPLv3 Windows Driver Loader</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="versionlbl">
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="exitbtn">
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="driverVersionlbl">
         <property name="text">
          <string>Driver Version:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="driverSizelbl">
         <property name="text">
          <string>Driver Size:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="driverFileTimelbl">
         <property name="text">
          <string>Driver File Time:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="driverVersiontxt">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(222, 222, 222);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="driverSizetxt">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(222, 222, 222);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="driverFileTimetxt">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(222, 222, 222);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="driverPathlbl">
         <property name="text">
          <string>Driver Path:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="driverPathtxt"/>
         </item>
         <item>
          <widget class="QPushButton" name="browsebtn">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="serviceNamelbl">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Service Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="serviceNametxt"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="displayNamelbl">
         <property name="text">
          <string>Display Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="serviceDisplayNametxt"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="serviceTypelbl">
         <property name="text">
          <string>Service Type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="serviceTypetxt"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="serviceStartlbl">
         <property name="text">
          <string>Service Start:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="serviceStarttxt"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="serviceErrorlbl">
         <property name="text">
          <string>Error:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="serviceErrortxt"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="registerbtn">
         <property name="text">
          <string>Register Service</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="unregisterbtn">
         <property name="text">
          <string>Unregister Service</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startbtn">
         <property name="text">
          <string>Start Service</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopbtn">
         <property name="text">
          <string>Stop Service</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
